frontend/src/components/Player.js

```diff
@@ -1,19 +1,18 @@
-import React, { Component } from "react";
+import React from "react";
 
-class Player extends Component {
-  render() {
-    return (
-      <span className="Player">
-        <h2>{this.props.name}</h2>
-        <img
-          src={require("./player1.jpg")}
-          alt={this.props.name}
-          height="100"
-          width="100"
-        />
-      </span>
-    );
-  }
-}
+const Player = (props) => {
```

Comment:
Player sounds like an entity, maybe chaaaaaange to playerCard?

Reply:
bd0d2c69a06b6a971340be0b04c5d1c23b69cc98